.github/workflows/benchmark.yml

```diff
@@ -86,8 +90,20 @@ jobs:
             **PR**: ${{ needs.benchmark.outputs.PR-BENCH-20 }}
             **MAIN**: ${{ needs.benchmark.outputs.MAIN-BENCH-20 }}
             
-      - uses: actions-ecosystem/action-remove-labels@v1
+  remove-label:
+    if: "always()"
+    needs: 
+      - benchmark
+      - output-benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove benchmark label
+        uses: octokit/request-action@v2.x
```

Comment:
actions-ecosystem/action-remove-labels was not good?

Reply:
it is using node12
https://github.com/actions-ecosystem/action-remove-labels/blob/d05162525702062b6bdef750ed8594fc024b3ed7/action.yml#L24

So in the future, github will run node12 actions in node16 runtimes. You usually get the deprecation notice.